README.md

```diff
@@ -214,30 +380,11 @@ There are two use cases for deep linking that can be relevant here:
 
 In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here:
```

Comment:
This wording feels awkward. How about this?

You need to configure deep links in your app for them to work. The configuration process for Klaviyo is no different than what is required for handling deep linking in general, so you can follow the [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) for deep linking in conjunction with the steps outlined here:

Reply:
Thanks for the suggestion. Done.